docs/production-install.md

```diff
@@ -12,12 +12,12 @@ The below sections will cover how to install the component into Kubernetes.
 
 #### Recommended prerequisites
 
-* Cert manager installed (https://hub.helm.sh/charts/jetstack/cert-manager)
-* gRPC compatible ingress controller installed for gRPC ingress (such as https://github.com/kubernetes/ingress-nginx)
-* Redis installed (https://github.com/helm/charts/tree/master/stable/redis-ha)
-* Optionally install NATS streaming server (helm chart: https://github.com/nats-io/k8s/tree/master/helm/charts/stan, additional docs: https://docs.nats.io/nats-on-kubernetes/minimal-setup)
+* Cert manager installed [https://hub.helm.sh/charts/jetstack/cert-manager](https://hub.helm.sh/charts/jetstack/cert-manager)
```

Comment:
This link doesn't work.  It looks like the cert-manager is no longer a valid link.  

Maybe this link is better to use:

https://cert-manager.io/docs/installation/helm/#installing-with-helm